src/Components/AvailableUser/AvailableUser.Component.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+import './AvailableUser.scss';
+
+class AvailableUser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      position: null
+    };
+  }
+
+  /**
+   * Method to handle position change
+   */
+  handlePositionChange = (e, i, value) => {
+    this.setState({ position: value });
+    let { id, index } = this.props;
+    this.props.handlePositionChange(value, id, index);
+  };
+
+  /**
+   * Method to handle user removal
+   */
+  handleRemove = () => {
+    let { id, index } = this.props;
+    this.props.handleRemove(id, index);
+  };
+
+  render() {
+    const { imageUrl, username, role, country } = this.props;
+    const styles = {
+      fontSize: '1.2rem',
+      width: '15vw',
+      height: '5vh',
+      backgroundColor: '#ffffff'
+    };
+    return (
+      <div className="user-card">
+        <img className="user-img" src={imageUrl} />
+        <div className="username">{username}</div>
+        <div className="user-role">
+          <span>{role}</span>
+        </div>
+        <div className="user-country">{country}</div>
+        <div className="change-role">
+          <SelectField
+            underlineStyle={{ display: 'none' }}
+            iconStyle={{ display: 'none' }}
+            labelStyle={{
+              textAlign: 'center',
+              marginRight: '1vw',
+              paddingRight: '0',
+              color: '#49abb0',
+              fontFamily: 'DIN Pro Extra Light',
+              fontWeight: 'normal',
```

Comment:
Can we put these styles as variables then use them here. Doesn't look very good here.

Reply:
Done.